admin/services/data-access/pin-generation.data.service.js

```diff
@@ -39,11 +39,22 @@ const serviceToExport = {
     return sqlService.readonlyQuery(sql, [param])
   },
 
+  /**
+   * @typedef CreateCheck
+   * @property {number} pupil_id
+   * @property {number} checkForm_id
+   * @property {number} checkWindow_id
+   * @property {boolean} isLiveCheck
+   * @property {import('moment').Moment} pinExpiresAt
```

Comment:
Is it meant to be like this?

Reply:
yes, but only in places where moment is not declared/required in the file itself.  There were a couple of these instances where `moment-timezone` was required at the top of the file, but `tsc` couldn't infer the `Moment` type from it.   When `moment` is imported, i simply use `{moment.Moment}`